solr/solr-ref-guide/modules/deployment-guide/pages/metrics-reporting.adoc

```diff
@@ -107,771 +93,287 @@ RequestHandlers can be configured to roll up core level metrics to the node leve
-----
-
-== Core Level Metrics
-
-These metrics are available only on a per-core basis.
-Metrics can be aggregated across cores using Shard and Cluster reporters.
+[source,text]
+http://localhost:8983/solr/admin/metrics?name=solr_core_requests_total&core=foobar_shard1_replica_n1
 
-=== Index Merge Metrics
+Request only the core index size `solr_core_index_size_bytes` metrics from collections labeled `foo` and `bar`:
 
-These metrics are collected under the `INDEX` category and track flush operations (documents being written to disk) and merge operations (segments on disk being merged).
+[source,text]
+http://localhost:8983/solr/admin/metrics?name=solr_core_index_size_bytes&collection=foo,bar
 
-For merge metrics, metrics are tracked with the distinction of "minor" and "major" merges (as merges with fewer documents will be typically more frequent).
-This is indicated by the `merge_type` label for the metric. The threshold for when a merge becomes large enough to be considered major is configurable, but
-defaults to 524k documents.
+== OTLP
 
-Metrics collection for index merges can be configured in the `<metrics>` section of `solrconfig.xml` as shown below:
+For users who do not use or support pulling metrics in Prometheus format with the `/admin/metrics` API, Solr also supports pushing metrics natively with https://opentelemetry.io/docs/specs/otlp/[OTLP], which is a vendor-agnostic protocol for pushing metrics via gRPC or HTTP. 
 
-[source,xml]
-----
-<config>
-  ...
-  <indexConfig>
-    <metrics>
-      <long name="majorMergeDocs">524288</long>
-    </metrics>
-    ...
-  </indexConfig>
-...
-</config>
-----
+OTLP is widely supported by many tools, vendors, and pipelines. See the OpenTelemetry https://opentelemetry.io/ecosystem/vendors/[vendors list] for more details on available and compatible options.
 
+=== OTLP properties
 
-== Metrics API
+Solr's internal OTLP exporter is disabled by default and is packaged with the OpenTelemetry module.
 
-The `admin/metrics` endpoint provides access to all the metrics for all metric groups.
+The module can be enabled with either the system property `-Dsolr.modules=opentelemetry` or the environment variable `SOLR_MODULES=opentelemetry`, similar to distributed tracing.
 
-A few query parameters are available to limit your request to only certain metrics:
+The OTLP exporter can be configured with the supported system properties below. These can also be set as environment variables by following these mapping rules:
 
-`group`::
-+
-[%autowidth,frame=none]
-|===
-|Optional |Default: `all`
-|===
-+
-The metric group to retrieve.
-The value `all` retrieves all metrics for all groups.
-Other possible values are: `jvm`, `jetty`, `node`, and `core`.
-More than one group can be specified in a request; multiple group names should be separated by a comma.
+- Replace `.` with `_`
+- Convert camelCase to UPPER_SNAKE_CASE
```

Comment:
oh really?  Where is this conversion happening?  I don't believe EnvUtils does this.